backend/airweave/platform/configs/auth.py

```diff
@@ -149,3 +149,23 @@ class GitHubAuthConfig(AuthConfig):
         title="Repository Name",
         description="Repository to sync in owner/repo format (e.g., 'airweave-ai/airweave')",
     )
+
+
+class DropboxAuthConfig(AuthConfig):
+    """Dropbox authentication credentials schema."""
+
+    client_id: str = Field(
+        title="Client ID", description="The OAuth client ID for your Dropbox app"
+    )
+    client_secret: str = Field(
+        title="Client Secret", description="The OAuth client secret for your Dropbox app"
+    )
+
+
+class GoogleAuthConfig(AuthConfig):
+    """Google authentication credentials schema."""
+
+    client_id: str = Field(title="Client ID", description="The OAuth client ID for your Google app")
+    client_secret: str = Field(
+        title="Client Secret", description="The OAuth client secret for your Google app"
+    )
```

Comment:
let's also set up auth configs for the onedrive, trello, and linear integrations